### DISCUSSION TITLE
Airflow does not seem to provide an API for log information. Is there any other way for me to obtain worker logs? It seems impossible to get the worker's log via http

### DISCUSSION THREAD

**Discussion by LinYuHeng-Con** [apache/airflow#27006] [Q&A]
Airflow does not seem to provide an API for log information. Is there any other way for me to obtain worker logs? It seems impossible to get the worker's log via http

**Comment by potiuk** [comment]
Just read the logs from the filesystem where you put them. Using Filesystem API of yours (ls, get fille etc.) combined with the way how you can configure airflow to use a number of logging frameworks (elasticsearch, s3, cloudwatch, stackdriver) https://airflow.apache.org/docs/apache-airflow-providers/core-extensions/logging.html - is enough to use the existing APIs. Rather than providing a common "logging" API, Airflow lets you plug-in into the existing logging frameworks that provide all capabilities you might need.
